lib/version.js

```diff
@@ -33,6 +36,15 @@ program
   })
   .parse(process.argv);
 
+if (!hasVersionFile) {
+  console.error(
+    `${chalk.yellow(
+      'No version.js file founded!'
```

Comment:
Should it be `versions.js` or `version.js`? The code checks for `versions.js` but the error message suggests `version.js`.

Grammatical nit - The error message should be "No versions.js file found" not founded.

Could you also shift the `const hasVersionFile` declaration to just above the check here? We would like to co-locate the variables to where they are being referenced, especially when this file is only being referenced once.